File: src/main/java/com/kindergarten/dance/controllers/MainController.java
package com.kindergarten.dance.controllers;

import com.kindergarten.dance.model.IndexSettings;
import com.kindergarten.dance.services.CoachService;
import com.kindergarten.dance.services.IndexSettingsService;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;
import org.springframework.ui.Model;

@Service
public class MainController {

    @Autowired
    private IndexSettingsService indexSettingsService;

    @Autowired
    private CoachService coachService;

    public void addModel(Model model) {
        IndexSettings settings = indexSettingsService.getFirst();
        model.addAttribute("phone", settings.getPhone());
        model.addAttribute("address", settings.getAdress());
        model.addAttribute("coaches", coachService.getAllActive());
    }

}

File: src/main/java/com/kindergarten/dance/controllers/MainController.kt
package com.kindergarten.dance.controllers

import com.kindergarten.dance.services.*
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.stereotype.Service
import org.springframework.ui.Model

@Service
class MainController {
    @Autowired
    lateinit var indexSettingsService: IndexSettingsService
    @Autowired
    lateinit var videoService: VideoService
    @Autowired
    lateinit var pagesService: PagesService

    fun addModel(model: Model) {
        val settings = indexSettingsService.first
        model.addAttribute("phone", settings.phone)
        model.addAttribute("address", settings.adress)
        model.addAttribute("coaches", pagesService.getAllActiveCoaches())
        model.addAttribute("ourHall", pagesService.getAllActiveOurHall())
        model.addAttribute("news", pagesService.getAllActiveNewslastTwo())
        model.addAttribute("inspirations",  pagesService.getAllActiveInspirations())
        model.addAttribute("costReq", pagesService.findAllActiveCostumeRequirements())
        model.addAttribute("video", videoService.findAllActive())
    }
}